lib/database_connection.rb

```diff
@@ -0,0 +1,17 @@
+require 'pg'
+
+class DatabaseConnection
+
+  def self.setup(database_name)
+    @connection = PG.connect(dbname: database_name)
+  end
+
+  def self.connection
+    @connection
+  end
+
+  def self.query(sql)
+    @connection.exec(sql)
+  end
+
+end
```

Comment:
It's very good how you set up the connection to the database within methods for cleaner code